Submitted By: Jim Gifford (jim at linuxfromscratch dot org)
Date: 2004-01-30
Initial Package Version: 2.6
Origin: Based on lfs-hackers 
Description: Fixes Compile with the 2.6 Kernel
 
$LastChangedBy$
$Date$

diff -Naur inetutils-1.4.2.orig/libinetutils/revoke.c inetutils-1.4.2/libinetutils/revoke.c
--- inetutils-1.4.2.orig/libinetutils/revoke.c	2002-06-26 03:15:06.000000000 +0000
+++ inetutils-1.4.2/libinetutils/revoke.c	2004-01-30 20:20:22.000000000 +0000
@@ -1,5 +1,7 @@
 /* stub revoke */
 
+#include <linux/version.h>
+
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
@@ -10,6 +12,7 @@
 extern int errno;
 #endif
 
+#if LINUX_VERSION_CODE < KERNEL_VERSION(2, 6, 0)
 int
 revoke (char *path)
 {
@@ -21,3 +24,5 @@
 #endif
   return -1;
 }
+#endif
+
